app/assets/javascripts/components/Register.jsx

```diff
@@ -0,0 +1,107 @@
+class Register extends React.Component {
+
+  constructor(props) {
+    super();
+    this.state = {"success": null};
+  }
+
+  FieldGroup({ id, label, help, ...props }) {
```

Comment:
this function is pretty generic, probably doesn't belong as a class method of this component. We'd prefer to put this into some kind of forms util file i think.

Reply:
Is there an existing file where I should put this?